tests/test_architectures.py

```diff
@@ -273,13 +319,13 @@ def test_complex():
         v_leak=np.array([0, 0, 0]),
         v_threshold=np.array([1, 1, 1]),
     )
-    # TODO: This should be a node that accepts two inputs
+    # TODO: This should be a node that accepts two input_type
     d = nir.Affine(
         weight=np.array([[[1, 3], [2, 3], [1, 4]], [[2, 3], [1, 2], [1, 4]]]),
         bias=np.array([0, 0]),
     )
     e = nir.Affine(weight=np.array([[1, 3], [2, 3], [1, 4]]), bias=np.array([0, 0]))
-    # TODO: This should be a node that accepts two inputs
```

Comment:
should I implement this? (ie with a simple NIR graph that takes two inputs and puts them through different Affine nodes) - or what's the plan with it?

Reply:
I reckon the plan was to provide a node that actually uses two different inputs, instead of having two inputs that connect to the same "port".

I suggest we merge it for now, but leave the TODOs for later :)